File: accounts/src/main/java/br/com/netlit/accounts/infra/error/SimpleErrorAttributes.java
package br.com.netlit.accounts.infra.error;

import org.springframework.boot.web.servlet.error.DefaultErrorAttributes;
import org.springframework.stereotype.Component;
import org.springframework.web.context.request.WebRequest;

import java.util.Map;

@Component
public class SimpleErrorAttributes extends DefaultErrorAttributes {

  @Override
  public Map<String, Object> getErrorAttributes(final WebRequest webRequest, final boolean includeStackTrace) {
    final Map<String, Object> errorAttributes = super.getErrorAttributes(webRequest, includeStackTrace);
    errorAttributes.remove("path");
    errorAttributes.remove("status");
    errorAttributes.remove("timestamp");
    return errorAttributes;
  }
}

File: payments/src/main/kotlin/netlit/payments/infra/error/SimpleErrorAttributes.kt
package netlit.payments.infra.error

import org.springframework.boot.web.servlet.error.DefaultErrorAttributes
import org.springframework.stereotype.Component
import org.springframework.web.context.request.WebRequest

@Component
class SimpleErrorAttributes : DefaultErrorAttributes() {

    override fun getErrorAttributes(webRequest: WebRequest, includeStackTrace: Boolean): Map<String, Any> {
        val errorAttributes = super.getErrorAttributes(webRequest, includeStackTrace)
        errorAttributes.remove("path")
        errorAttributes.remove("status")
        errorAttributes.remove("timestamp")
        return errorAttributes
    }
}